molecule_hetznercloud/cookiecutter/scenario/driver/hetznercloud/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/destroy.yml

```diff
@@ -37,6 +37,30 @@
       retries: 300
       with_items: "{{ server.results }}"
 
+    - pause:
```

Comment:
I think this probably shouldn't be required although I understand why you did it. I would consider it a bug upstream if the destroy task doesn't handle waiting for volumes to get torn down. I would instead raise a ticket over at https://github.com/hetznercloud/hcloud-python (the underlying Python implementation) where I do often and they are actually very responsive. That way we more reliable upstream which I generally think is worth the effort. Let me know what you think.

Reply:
Sure, I'll try to reproduce the issue again and create an issue upstream.

Reply:
Hmm not able to reproduce anymore... I've removed the pause task